src/robot/client.cpp

```diff
@@ -5,7 +5,6 @@
 #include <grpcpp/support/status.h>
 #include <robot/v1/robot.grpc.pb.h>
 #include <robot/v1/robot.pb.h>
-#include <rpc/rpc.h>
```

Comment:
I'm not sure why this include was here, as it seems to be part of the old Sun RPC system. Not all systems seem to offer it in the usual places or default packages, so if we can get away without it I think that would be best.

Reply:
Just tested to confirm, this is unnecessary. Most likely just an artifact of old testing. Please feel free to just delete it, or I can do so in a future PR!

Reply:
I'll just drop it out in this PR since I already have the change.